### PR TITLE
Mark more texts for localization

### PR DESCRIPTION
**Pull request by quiple**
I've separated out the content of #2462 into this PR that I think is not controversial.

* Makes more text available for localization.
* Corrects the reversal of `accessibilityLabel` and `accessibilityHint`.
* Adds or modifies the content of some `accessibilityLabel` and `accessibilityHint`.
* Adds the missing period in `LanguageSettings.tsx`.

Thank you!

### CODE REVIEW

File: src/view/com/modals/LinkWarning.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export function Component({text, href}: {text: string; href: string}) {
             onPress={onPressVisit}
             accessibilityLabel={_(msg`Visit Site`)}
             accessibilityHint=""
```
Comment (surfdude29) [inline src/view/com/modals/LinkWarning.tsx:76]: Sorry to jump in without invitation, and I know this isn't strictly related to localisation, what about adding the missing accessibility hint here, something like "Opens the linked website"?
```suggestion
            accessibilityHint={_(msg`Opens the linked website`)}
```

File: src/view/com/modals/LinkWarning.tsx [hunk 2]
```diff
@@ -86,7 +86,7 @@ export function Component({text, href}: {text: string; href: string}) {
             }}
             accessibilityLabel={_(msg`Cancel`)}
             accessibilityHint=""
```
Comment (surfdude29) [inline src/view/com/modals/LinkWarning.tsx:88]: And here, what about "Don't open the linked website"?
```suggestion
            accessibilityHint={_(msg`Don't open the linked website`)}
```

File: src/view/screens/Moderation.tsx [hunk 3]
```diff
@@ -69,8 +69,8 @@ export function ModerationScreen({}: Props) {
           style={[styles.linkCard, pal.view]}
           onPress={onPressContentFiltering}
           accessibilityRole="tab"
-          accessibilityHint="Content filtering"
-          accessibilityLabel="">
+          accessibilityLabel={_(msg`Content filtering`)}
+          accessibilityHint="">
```
Comment (surfdude29) [inline src/view/screens/Moderation.tsx:73]: Here, what about "Opens the content filtering preferences"?
```suggestion
          accessibilityHint={_(msg`Opens the content filtering preferences`)}>
```

File: src/view/screens/Settings/index.tsx [hunk 4]
```diff
@@ -730,7 +730,9 @@ export function SettingsScreen({}: Props) {
           onPress={() => openModal({name: 'change-password'})}
           accessibilityRole="button"
           accessibilityLabel={_(msg`Change password`)}
-          accessibilityHint={_(msg`Change your Bluesky password`)}>
+          accessibilityHint={_(
+            msg`Opens modal for change your Bluesky password`,
```
Comment (surfdude29) [inline src/view/screens/Settings/index.tsx:734]: I think "for changing" would work better here?
```suggestion
            msg`Opens modal for changing your Bluesky password`,
```

File: src/view/screens/Settings/index.tsx [hunk 5]
```diff
@@ -752,7 +754,7 @@ export function SettingsScreen({}: Props) {
           accessibilityRole="button"
           accessibilityLabel={_(msg`Export my data`)}
           accessibilityHint={_(
-            msg`Download Bluesky account data (repository)`,
+            msg`Opens modal for download Bluesky account data (repository)`,
```
Comment (surfdude29) [inline src/view/screens/Settings/index.tsx:757]: Similar here, I think "for downloading" would work better?
```suggestion
            msg`Opens modal for downloading Bluesky account data (repository)`,
```

**Comment by quiple** [comment]
@surfdude29 Thank you! It looks much better.

**Comment by surfdude29** [comment]
@quiple No worries, happy to help :)

**Comment by surfdude29** [comment]
I found one more string that you might like to add this PR, `Select languages` here:

https://github.com/bluesky-social/social-app/blob/1b992d5d6b753228a4fe5b2b42042873071bbcc3/src/view/screens/LanguageSettings.tsx#L299

**Comment by surfdude29** [comment]
I noticed that `followers` and `posts` in the profile headers aren't localised:

https://github.com/bluesky-social/social-app/blob/7390863a1005eeadbb6dbdcbc47f9cc13298e101/src/view/com/profile/ProfileHeader.tsx#L575-L580

https://github.com/bluesky-social/social-app/blob/7390863a1005eeadbb6dbdcbc47f9cc13298e101/src/view/com/profile/ProfileHeader.tsx#L603-L608

Does this require #2382 ?

**Comment by quiple** [comment]
@surfdude29 Yes, I think so.

File: src/view/screens/Settings/index.tsx [hunk 6]
```diff
@@ -531,8 +531,8 @@ export function SettingsScreen({}: Props) {
           ]}
           onPress={openFollowingFeedPreferences}
           accessibilityRole="button"
-          accessibilityHint=""
-          accessibilityLabel={_(msg`Opens the home feed preferences`)}>
+          accessibilityLabel={_(msg`Home feed preferences`)}
```
Comment (surfdude29) [inline src/view/screens/Settings/index.tsx:534]: As a result of #2965 `Home` to `Following` should be updated here:

```suggestion
          accessibilityLabel={_(msg`Following feed preferences`)}
```

File: src/view/screens/Settings/index.tsx [hunk 7]
```diff
@@ -531,8 +531,8 @@ export function SettingsScreen({}: Props) {
...
+          accessibilityHint={_(msg`Opens the home feed preferences`)}>
```
Comment (surfdude29) [inline src/view/screens/Settings/index.tsx:535]: And here too

```suggestion
          accessibilityHint={_(msg`Opens the Following feed preferences`)}>
```

**Comment by quiple** [comment]
@pfrazee I think it's okay to merge this PR right away, but what do you think?

**Review by pfrazee — APPROVED** [review]
LGTM! Thank you everybody!

**Pull request merged**